AWSPinpointTests/AWSPinpointTargetingClientTests.m

```diff
@@ -55,10 +55,10 @@ - (void)setUp {
     [[NSUserDefaults standardUserDefaults] removeSuiteNamed:@"AWSPinpointTargetingClientTests"];
     self.userDefaults = [[NSUserDefaults alloc] initWithSuiteName:@"AWSPinpointTargetingClientTests"];
 
-    [self initializeMockApplicationWithOptOut:YES];
-    [self initializePinpointWithConfiguration:[self getDefaultAWSPinpointConfiguration] forceCreate:NO];
+    //[self initializePinpointWithConfiguration:[self getDefaultAWSPinpointConfiguration] forceCreate:NO];
     [self.userDefaults removeObjectForKey:@"AWSPinpointEndpointAttributesKey"];
     [self.userDefaults removeObjectForKey:@"AWSPinpointEndpointMetricsKey"];
```

Comment:
we should make these constants too, but not a big deal

Reply:
nice catch, i'll do this